weave-js/src/errors.ts

```diff
@@ -1 +1,23 @@
+import {ApolloError} from '@apollo/client';
+import {get} from 'lodash';
+
 export class UseNodeValueServerExecutionError extends Error {}
+
+export function extractErrorMessageFromApolloError(
```

Comment:
indeed. I don't use the "extra err msg" one in this PR but i brought it over since it's prob useful for future.

if we wanted to invest the time, we could update all the imports in core and delete these utils there. my only concern is I set up weavejs with a newer (major) version of apollo than core, so it might be a good idea to keep separate copies of these utils in case they actually need to diverge for some reason?